### PR TITLE
[otbn, dv] Enables secure wipe

### PR DESCRIPTION
**Pull request by prajwalaputtappa**
This commit adds necessary changes to enable security wipe and fixes the
wdr register commit issue when secure wipe is done. It also fixes rtl
and iss trace entry mismatch during coverage collection.

Signed-off-by: Prajwala Puttappa <prajwalaputtappa@lowrisc.org>

### CODE REVIEW

File: hw/ip/otbn/dv/model/otbn_trace_checker.cc [hunk 1]
```diff
@@ -246,10 +246,12 @@ bool OtbnTraceChecker::MatchPair() {
     return false;
   }
 
-  // We've got a matching pair. Move the ISS data out of the (now defunct)
+  // We've got a matching insn pair. Move the ISS data out of the (now defunct)
```
Comment (rswarbrick) [inline hw/ip/otbn/dv/model/otbn_trace_checker.cc:249]: Ooh, probably "insn" is wrong here (because of the secure wipe U/V trace entries). Maybe drop this, or change to something like "matching pair of entries"?

File: hw/ip/otbn/dv/uvm/tb.sv [hunk 2]
```diff
@@ -25,7 +25,7 @@ module tb;
   wire [NUM_MAX_INTERRUPTS-1:0] interrupts;
 
   // Configure internal secure wipe
-  localparam bit SecWipeEn  = 1'b0;
+  localparam bit SecWipeEn  = 1'b1;
```
Comment (rswarbrick) [inline hw/ip/otbn/dv/uvm/tb.sv:28]: Probably best to do this as a separate commit to make bisection a bit clearer (although I'm happy for it to be in the same PR)

Comment (prajwalaputtappa) [reply hw/ip/otbn/dv/uvm/tb.sv:28]: Split it into a different commit in the same PR.
